osu.Server.Spectator/Database/DatabaseAccess.cs

```diff
@@ -154,22 +156,40 @@ public async Task UpdateRoomParticipantsAsync(MultiplayerRoom room)
             }
         }
 
-        public async Task ClearRoomScoresAsync(MultiplayerRoom room)
+        public async Task<multiplayer_playlist_item> GetPlaylistItemFromRoomAsync(long roomId, long playlistItemId)
```

Comment:
If `QueryFirstOrDefault`, then
```suggestion
        public async Task<multiplayer_playlist_item?> GetPlaylistItemFromRoomAsync(long roomId, long playlistItemId)
```